pages/03.guides/02.settings/default.md

```diff
@@ -19,15 +19,15 @@ admin:
 ### General Tab 
 There are a multitude of settings that can be changed. The Port and Logging Level require a manual restart of the server to take effect. The cache directory is where temporary files will be placed, for example, when a user reads a file, the file is usually cached or the output of processing is placed in this directory. Kavita will clear this out regularly.
 
-Allowing Anonymous Usage Collection is here, you can opt out. This setting allows you to send Anonymous data about your install which helps the Kavita team understand it's user base to make design decisions and performance enhancements. Some information that is collected is installId (helps us track how many installs we have), OS, version, if you are using docker or not and a few small pieces of data for features. None of the data can be tracked back to you. You can find the list [here](https://wiki.kavitareader.com/en/faq). The Stat collection is within the Kavita and KavitaStats repositories.
+Anonymous Usage Collection is here. You can opt out by turning off "Send Data" from the Admin Dashboard. This setting allows you to send Anonymous data about your install which helps the Kavita team understand its user base to make design decisions and performance enhancements. Some information that is collected is file types within Kavita, number of users, screen sizes and browser information, OS, version, and if you are using Docker or not. The Stat collection is within the Kavita and KavitaStats repositories.
```

Comment:
Can we reword the stat collection to match what we say on the nightly? We don't collect screen size or browser information, but we do collect number of genres, number of files, if certain features are used (see Kavita stats or we should link like in FAQ).

I'm pretty sure FAQ has updated description of what we collect.